src/Microsoft.PowerShell.Commands.Utility/commands/utility/RemoveAliasCommand.cs

```diff
@@ -0,0 +1,117 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// The implementation of the "remove-alias" cmdlet
+    /// </summary>
+    ///
+    [Cmdlet(VerbsCommon.Remove, "Alias", DefaultParameterSetName = "Default", HelpUri = "")]
```

Comment:
@daxian-dbw @HemantMahawar @SteveL-MSFT  We need new HelpUri.

Reply:
Issue opened: #15625